src/components/Profile/Profile.js

```diff
@@ -5,10 +5,12 @@ import MyPostsContainer from './MyPosts/MyPostsContainer';
 import ProfileInfo from './ProfileInfo/ProfileInfo';
 
 const Profile = (props) => {
-  const { profile, status, updateStatus } = props;
+  const {
+    profile, status, updateStatus, isOwner, savePhoto, saveProfile,
+  } = props;
   return (
     <div>
-      <ProfileInfo profile={profile} status={status} updateStatus={updateStatus} />
+      <ProfileInfo profile={profile} isOwner={isOwner} status={status} savePhoto={savePhoto} saveProfile={saveProfile} updateStatus={updateStatus} />
```

Comment:
In case the linter doesn't mind it - it's better to keep the imported things each on a new line.
Take a look at ProfileContainer.js an the <Profile .../>.